.github/workflows/scripts/windows-container-prep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# abstract the prepare commands for the windows build into a script that can be reused
+# instead of a yaml file
+cd /win
+ln -s /__w/subsurface/subsurface .
+ls -l
+ls -l subsurface/scripts
+echo "installing missing container components"
+apt-get install -y ca-certificates libtool
```

Comment:
BTW: that was a hack because for some reason right when I worked on this new certificates were pushed and things were failing in the container. And then when I realized I had missed libtool I added this here as well.
The certificate thing might be needed - don't know. But libtool could certainly be fixed in the base container.

Reply:
Will tidy that up in the base container then.